### PR TITLE
Start integrating app overrides

### PR DESCRIPTION
**Pull request by bitjson**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,10 +7,12 @@
...
     "start": "echo && echo \"Choose a distribution by running 'npm run apply:copay' or 'npm run apply:bitpay'.\" && echo",
-    "apply:copay": "npm i fs-extra@0.30 && cd app-template && node apply.js copay && cd .. && npm i",
-    "apply:bitpay": "npm i fs-extra@0.30 && cd app-template && node apply.js bitpay && cd .. && npm i",
+    "apply:copay": "npm run before-install && cd app-template && node apply.js copay && cd .. && npm i",
+    "apply:bitpay": "npm run before-install && cd app-template && node apply.js bitpay && cd .. && npm i",
     "unstage-package": "git reset package.json",
-    "clean-all": "git clean -dfx"
+    "clean-all": "git clean -dfx",
+    "before-install": "npm i fs-extra@0.30 node-version-check && npm run require-node-version-6",
+    "require-node-version-6": "node-version-check '>=6'"
```
Comment (ajp8164) [inline package.json:15]: Seems like "require-node-version-6" should be just "require-node-version" (then it doesn't change when we require version 7.

Comment (bitjson) [reply package.json:15]: I put the number in that script name so it's a bit clearer when you see the error message. (Unfortunately the error is a bit overwhelming if you don't know what you're looking at.) Now even if you don't read it, it's hard to miss the problem.

Comment (bitjson) [reply package.json:15]: This is the only file that uses the script name, so when we upgrade, we probably just want to change all the `6`s to `8`s or whatever

Comment (ajp8164) [reply package.json:15]: ah, no echo

**Pull request merged**